docs/development/DEPLOYMENT.md

```
@@ -0,0 +1,118 @@
+
+- `EXPO_ANDROID_SERVICE_ACCOUNT_KEY_PATH`: Path to your Google Cloud service account JSON key.
+
+### Build Profiles
+
+The `eas.json` file defines three build profiles:
+
+- **`development`**: Used for creating development clients. It's configured to build an internal distribution with a development client enabled. For Android, it builds an `.apk` file.
+- **`preview`**: Used for creating preview builds for internal testing. It's configured for internal distribution and includes simulator builds for iOS. For Android, it also builds an `.apk` file.
+- **`production`**: Used for creating production builds for App Store and Google Play.
+
+## EAS Build Setup
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)